### PR TITLE
[clipboardxx] Add new port

### LINKED ISSUE #23938: [New Port Request] ClipboardXX
Library name: ClipboardXX
Library description: Header only, lightweight and cross platform C++ library for copy and paste text from clipboard.
Source repository URL: https://github.com/Arian8j2/ClipboardXX

Kindly extract my `clipboardxx.zip` archive into the `ports` directory and test whether my port is fine:
[clipboardxx.zip](https://github.com/microsoft/vcpkg/files/8402106/clipboardxx.zip)

Then consider committing it to the master branch since my fork is conflicted. Thank you very much. 👍

### PR DESCRIPTION
**Pull request by Pospelove**
fixes #23938

### CODE REVIEW

**Comment by Pospelove** [comment]
@JonLiu1993 sorry what kind of author response is required? 🤾

**Comment by Pospelove** [comment]
@JonLiu1993 There is no review! 🆘 😭  🧦

**Comment by JonLiu1993** [comment]
> @JonLiu1993 sorry what kind of author response is required? 🤾

@Pospelove, I've made a review suggestion that you can apply

**Comment by Pospelove** [comment]
@JonLiu1993 Maybe there is a review draft only visible to you?

**Review by JonLiu1993 — APPROVED** [review]

**Comment by strega-nil-ms** [comment]
This name is wrong - the port name should be `clipboardxx`, not `clipboardcxx`.

**Review by strega-nil-ms — CHANGES_REQUESTED** [review]
Two things need to be done:

- please change the name
- please use the CMake build

File: ports/clipboardcxx/vcpkg.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "clipboardcxx",
```
Comment (JonLiu1993) [inline ports/clipboardcxx/vcpkg.json:2]:
```suggestion
  "name": "clipboardxx",
```

**Comment by Pospelove** [comment]
@strega-nil-ms Thanks! Seems to be ok now

**Review by JonLiu1993 — APPROVED** [review]

**Review by strega-nil-ms — APPROVED** [review]

**Comment by strega-nil-ms** [comment]
Thanks!

**Pull request merged**